crates/framework/src/multiple_buffer.rs

```diff
@@ -209,6 +209,7 @@ mod tests {
             assert_eq!(*slot, 42);
         }
         {
+            #[allow(clippy::redundant_clone)]
```

Comment:
I would love to have a comment here that this can go away at some time with a link to the issue.

Reply:
Added comments to all three locations.